scripts/tag-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -eof pipefail
+
+if [ $# -eq 0 ]
+then
+    echo "Need to supply a release tag"
+    exit 0
+fi
+
+TAG=$1
+
+if $(git tag | grep -q $TAG); then
+    echo "Tag $TAG exists, replacing"
+    git tag --delete $TAG
+    git push --delete origin $TAG
+else
+    echo "New tag $TAG"
+fi
+
+git tag -s -a $TAG -i -m "Release $TAG"
```

Comment:
The `-i` switch is only available starting on version 2.12.0 of git, so in my Ubuntu system (which for some reason is fully updated and still has 2.7.4), if failed:

```
pedro@Ubuntu-Dev-PC:~/src/github.com/status-im/status-react$ scripts/tag-release.sh test-pp
New tag test-pp
error: unknown switch `i'
```

Reply:
Will try this on my system: http://lifeonubuntu.com/upgrading-ubuntu-to-use-the-latest-git-version/

Reply:
Interesting, see no reason why couldn't get rid of this. Do all the other tag operations work for you on Ubuntu and with that (LTS? git version)? I know there are some changes in recent-ish git versions related to pushing tags.

Reply:
Removed -i tag

Reply:
It would be nice to add a git version at the beginning of the script, and ask the dev to update git version if less than 2.12.0. Something like:

```shell
verlte() {
    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
}

verlt() {
    [ "$1" = "$2" ] && return 1 || verlte $1 $2
}

$gitVersion = $(git --version) | sed -e "s/^git version //"
verlt $gitVersion 2.12.0 || echo "Please upgrade git to latest version (>= 2.12.0)"
```

Reply:
Yeah, I'm on LTS.

Reply:
We could do that but I don't think this is some magic git-fu so old/LTS versions should be fine. Does it work for you with old git without the `-i` flag?